scss/components/_activity-panel.scss

```diff
@@ -1,13 +1,12 @@
 .civicase__activity-panel {
   margin-top: 37px;
-  width: 484px;
 
   .panel {
     overflow: auto;
-    width: 484px;
 
     &.affix {
       position: fixed !important;
+      width: calc(30% - 12px);
```

Comment:
Can we drop a little comment why we are subtracting `12px` specifically?

Reply:
@swastikpareek Honestly this is a magic number. I have tried to come up with a explanation but could not find one after spending 10-15 min. The panel has 30% width, but when its affixed, we get a slight jitter unless we specify `-12px`.